traffic_ops/testing/api/utils/utils.go

```diff
@@ -73,3 +76,58 @@ func Compare(t *testing.T, expected []string, alertsStrs []string) {
 		}
 	}
 }
+
+type CkReqFunc func(*testing.T, toclientlib.ReqInf, interface{}, error)
+
+func CkRequest(c ...CkReqFunc) []CkReqFunc {
```

Comment:
GoDoc comment?

Also though this is just an identity function - what purpose does it serve?